Common/Data/Consolidators/RangeBarConsolidator.cs

```diff
@@ -0,0 +1,84 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.Market;
+using System;
+
+namespace QuantConnect.Data.Consolidators 
+{
+    public class RangeBarConsolidator : DataConsolidator<Tick>
+    {
+        private RangeBar _currentBar;
+        private readonly int _length;
+
+        /// <summary>
+        /// Creates a new instance of <see cref="RangeBarConsolidator"/> of given length (pips).
```

Comment:
If the length refers to pips, we shouldn't compare the prices (e.g., `price - Low > Length`), but the minimum price variation. Since we are comparing prices, it shouldn't be an integer.